src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+          method="post">
+
+      <div class="Box">
+
+        <div class="Header"> Personal Information</div>
+
+        <label class="form-field">
+        Surname:
+          <input type="text" 
+                 name="Surname" 
+                 class="field"
+                 required>
```

Comment:
```suggestion
          <input type="text" 
             name="Surname" 
             class="field"
             required
           >
```
неправильное форматирование